python/ray/scripts/scripts.py

```diff
@@ -323,6 +350,20 @@ def start(node_ip_address, redis_address, address, redis_port, port,
         if port is not None and port != redis_port:
             raise ValueError("Cannot specify both --port and --redis-port "
                              "as port is a rename of deprecated redis-port")
+    if include_webui is not None:
```

Comment:
If it's not None we should probably either error or set the value for include_dashboard to what include_webui is set to

Reply:
Similar to what you have for the host flags below

Reply:
Makes sense.